README.md

```diff
@@ -143,6 +143,13 @@ resource "lxd_container" "test1" {
   }
 }
 ```
+Importing existing containers is possible with the following ID syntax
+terraform import lxd_container.container [remote:]name[/images:alpine/3.5].
+where:
+
+remote, optional, is the name of the remote in the provider config
+name, required, is the container name
+/images:alpine/3.5, optional, translates to image = images:alpine/3.5 in the resource configuration
```

Comment:
I think this should be moved to the "Reference" area of the documentation rather than in the "Example" area. Perhaps a new section called "Importing" that is similar to [this](https://github.com/sl1pm4t/terraform-provider-lxd/blob/master/README.md#container-network-access).

Also, if you look at the rendered README, the last three lines are all placed on a single line. I think they should be formatted as a list in Markdown.